.github/workflows/nightly_test.yml

```diff
@@ -77,6 +77,35 @@ permissions:
   packages: write
 
 jobs:
+  Orca-Basic-Py37-Spark3:
+    if: ${{ github.event.schedule || github.event.inputs.artifact == 'Orca-ExampleTest-Tf1-Py37-Spark3' || github.event.inputs.artifact == 'all' }} 
+    runs-on: [self-hosted, Gondolin, ubuntu-20.04-lts]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK8
+      uses: ./.github/actions/jdk-setup-action
+    - name: Set up maven
+      uses: ./.github/actions/maven-setup-action
+    - name: Setup env
```

Comment:
Do we really need jdk and maven in nightly-test? (though they don't cost much time